extensions/panache/mongodb-panache/deployment/src/main/java/io/quarkus/mongodb/panache/deployment/PanacheResourceProcessor.java

```diff
@@ -50,13 +50,12 @@ void registerJsonbSerDeser(BuildProducer<JsonbSerializerBuildItem> jsonbSerializ
     }
 
     @BuildStep
-    void registerJacksonSerDeser(BuildProducer<JacksonModuleBuildItem> customSerDeser) {
-        customSerDeser.produce(
-                new JacksonModuleBuildItem.Builder("ObjectIdModule")
-                        .add(io.quarkus.mongodb.panache.jackson.ObjectIdSerializer.class.getName(),
-                                io.quarkus.mongodb.panache.jackson.ObjectIdDeserializer.class.getName(),
-                                ObjectId.class.getName())
-                        .build());
+    JacksonModuleBuildItem registerJacksonSerDeser() {
```

Comment:
Not that it really matters, but I would prefer this to be reverted. The reason is to keep it consistent with what happens for JSON-B.

Reply:
as you want, for me less code is better that's why I refactor it this way ...

Reply:
I agree in principal, but for me consistency is a hair more important :)

Reply:
OK done

Reply:
Thansk!